### PR TITLE
TEVA-1162 bold checkbox labels

### PR DESCRIPTION
**Pull request by alexbowen**
https://dfedigital.atlassian.net/browse/TEVA-1162

Screenshot 2020-08-24 at 14 39 28
Screenshot 2020-08-24 at 14 39 15

### CODE REVIEW

File: app/views/shared/_checkbox_group.html.haml [hunk 1]
```diff
@@ -5,9 +5,10 @@
       %input.govuk-input.search-input__input{ placeholder: "Search" }
       %button{type: "submit"}
         %i.fa.fa-search
-  = f.govuk_collection_check_boxes attribute_name,
-    options,
-    value_method,
-    text_method,
-    hint_method = hint_method,
-    small: small
+  %div{ class: "#{classes.present? ? classes : ''}" }
```
Comment (cpjmcquillan) [inline app/views/shared/_checkbox_group.html.haml:8]: You can give the classes attribute to the FormBuilder object itself, so no need for this line

File: app/views/pages/home/_filters.html.haml [hunk 2]
```diff
@@ -9,5 +9,5 @@
         = t("jobs.filters.#{filter_type}")
 
     .checkbox-group__container{ aria: { labelledby: "checkbox-group-heading-#{filter_type}" } }
-      = render 'shared/checkbox_group', search: false, f: f, attribute_name: filter_type.to_sym, options: @jobs_search_form.send("#{filter_type.chop}_options".to_sym), value_method: :first, text_method: :last, small: true
+      = render 'shared/checkbox_group', classes: '', search: false, f: f, attribute_name: filter_type.to_sym, options: @jobs_search_form.send("#{filter_type.chop}_options".to_sym), value_method: :first, text_method: :last, small: true
```
Comment (cpjmcquillan) [inline app/views/pages/home/_filters.html.haml:12]: I think you should be able to omit `classes: ''`, it should be ok for it to be nil

File: app/views/shared/_filter_group.html.haml [hunk 3]
```diff
@@ -59,7 +59,7 @@
             = group[:title]
 
       .govuk-accordion__section-content.filter-group__checkboxes{ id: "accordion-default-content-#{index}", aria: { labelledby: "accordion-default-heading-#{index}" }, data: { group: group[:key] } }
-        = render 'shared/checkbox_group', f: f,
+        = render 'shared/checkbox_group', classes: '', f: f,
```
Comment (cpjmcquillan) [inline app/views/shared/_filter_group.html.haml:62]: Same as above

File: app/views/pages/home/_filters.html.haml (hunk 2 above)
Comment (alexbowen) [reply app/views/pages/home/_filters.html.haml:12]: classes has to exist according to error when it is omitted

File: app/views/shared/_checkbox_group.html.haml [hunk 4]
```diff
@@ -10,4 +10,5 @@
     value_method,
     text_method,
     hint_method = hint_method,
-    small: small
+    small: small,
+    classes: classes
```
Comment (cpjmcquillan) [inline app/views/shared/_checkbox_group.html.haml:14]: What about if we do `classes: classes || ''` here instead?

Comment (alexbowen) [reply app/views/shared/_checkbox_group.html.haml]: `classes: (defined? classes) ? classes : ''`

**Review by cpjmcquillan — APPROVED** [review]

**Pull request merged**